driver/Dockerfile

```diff
@@ -8,7 +17,5 @@ LABEL name="IBM Spectrum Scale CSI driver" \
       description="CSI Plugin for IBM Spectrum Scale"\
       maintainers="IBM Spectrum Scale"
 COPY licenses /licenses
-
-COPY _output/ibm-spectrum-scale-csi /ibm-spectrum-scale-csi
-RUN chmod +x /ibm-spectrum-scale-csi
+COPY --from=builder /go/src/github.com/IBM/ibm-spectrum-scale-csi-driver/_output/ibm-spectrum-scale-csi /ibm-spectrum-scale-csi
```

Comment:
If the above is comment is wrong at line 4, we should update this path too...

Reply:
Yeah, we should.  I did that in my multi-arch build, but forgot to do it in this version.  Thanks for the catch!